server-side/DAL/IndexedDataS3PfsDal.ts

```
@@ -96,12 +110,73 @@ export class IndexedDataS3PfsDal extends AbstractS3PfsDal
 		}
 	}
 
+	async isObjectLocked(key: string){
+		const tableName = LOCK_ADAL_TABLE_NAME;
+		try
+		{
+			const lockAbsoluteKey = this.getAbsolutePath(key).replace(new RegExp("/", 'g'), "~");
+			const getHidden: boolean = true;
+			const lockRes: any = await this.getObjectFromTable(lockAbsoluteKey, tableName, getHidden);
+			lockRes.Key = this.getRelativePath(lockRes.Key.replace(new RegExp("~", 'g'), "/"));
+
+			return lockRes;
+		}
+		catch
+		{
```

Please add here comment to explain this empty catch- I assume that you returns null because the adal retunes "object not found" exception you catch here?